src/ProjectService.Validation/EditProjectValidator.cs

```diff
@@ -1,105 +1,155 @@
-                    .WithMessage(
-                        $"Document contains invalid path. Only such paths are allowed: {Paths.Aggregate((x, y) => x + ", " + y)}"))
-                .DependentRules(() =>
-                {
-                    When(x => GetOperationByPath(x, Name) != null, () =>
-                    {
-                        RuleFor(x => x.Operations).UniqueOperationWithAllowedOp(Name, "replace");
-
-                        RuleFor(x => (string)GetOperationByPath(x, Name).value)
-                            .NotEmpty()
-                            .MaximumLength(150)
-                            .WithMessage("First name is too long.");
-                    });
-
-                    When(x => GetOperationByPath(x, ShortName) != null, () =>
-                    {
-                        RuleFor(x => x.Operations).UniqueOperationWithAllowedOp(ShortName, "replace");
-
-                        RuleFor(x => (string)GetOperationByPath(x, ShortName).value)
-                            .NotEmpty()
-                            .MaximumLength(30)
-                            .WithMessage("Short name is too long.");
-                    });
-
-                    When(x => GetOperationByPath(x, Description) != null, () =>
-                    {
-                        RuleFor(x => x.Operations).UniqueOperationWithAllowedOp(Description, "replace");
-                    });
-
-                    When(x => GetOperationByPath(x, ShortDescription) != null, () =>
-                    {
-                        RuleFor(x => x.Operations).UniqueOperationWithAllowedOp(ShortDescription, "replace");
-
-                        RuleFor(x => (string)GetOperationByPath(x, ShortDescription).value)
-                            .MaximumLength(300)
-                            .WithMessage("Short description is to long");
-                    });
-
-                    When(x => GetOperationByPath(x, Status) != null, () =>
-                    {
-                        RuleFor(x => x.Operations).UniqueOperationWithAllowedOp(Status, "replace");
-
-                        RuleFor(x => (ProjectStatusType)GetOperationByPath(x, Status).value)
-                            .IsInEnum()
-                            .WithMessage("Wrong status value.");
-                    });
-
-                    When(x => GetOperationByPath(x, DepartmentId) != null, () =>
-                    {
-                        RuleFor(x => x.Operations).UniqueOperationWithAllowedOp(DepartmentId, "replace");
-
-                        RuleFor(x => GetOperationByPath(x, DepartmentId).value)
-                            .Must(x => Guid.TryParse(x.ToString(), out Guid _))
-                            .WithMessage("Wrong department id value.");
-                    });
-                });
+          return true;
         }
+
+        _logger.LogWarning(
+          "Can not find department with this id {departmentId}: " +
```

Comment:
write it in 1 string(without '+') + replace 'Environment.NewLine' with 'errors' +
string.Join('\n', response.Message.Errors) should be third parameter - after departmentId.

Reply:
done